package.json

```diff
@@ -87,7 +87,7 @@
     "node": "16"
   },
   "dependencies": {
-    "@dialpad/dialtone-icons": "0.0.17",
+    "@dialpad/dialtone-icons": "latest",
```

Comment:
Hmm I know it is annoying to update this manually all the time, but is it safe to do this? especially since this is a non-dev dependency?

It's possible we could have a bunch of stuff in staging, update dialtone-icons with a breaking change, and then release dialtone-vue unknowingly with this broken change in it. Also how does this work if a project has included dialtone-vue as a dependency? dialtone-icons would just update to the latest version when they npm install in product potentially breaking something?

Reply:
I don't think so, this is just to avoid npm errors between 'vue3' and 'latest' tags.

You still need to do `npm update @dialpad/dialtone-icons` to update this to latest version of that tag. If you do `npm install` it doesn't update the current version you have, but prevents `vue3` tag version updating to `latest` and breaking everything cause vue2/vue3 dependencies.